sdk/translation/azure-ai-translation-document/azure/ai/translation/document/models/_patch.py

```diff
@@ -7,9 +7,472 @@
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
 
-from typing import List
+from typing import Any, List, Mapping, Optional, Union, overload, TYPE_CHECKING
+import datetime
+from ._models import (
+    DocumentStatus as GeneratedDocumentStatus,
+    TranslationStatus as GeneratedTranslationStatus,
+    TranslationGlossary as GeneratedTranslationGlossary,
+    TranslationTarget as GeneratedTranslationTarget,
+    BatchRequest,
+    SourceInput,
+    DocumentFilter,
+)
+from ._enums import StorageInputType
 
-__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from .. import models as _models
+
+
+def convert_status(status, ll=False):
+    if ll is False:
+        if status == "Cancelled":
```

Comment:
We could probably make these checks case insensitive:
`if status.lower() == 'cancelled'`

Reply:
Updated

Reply:
Updated